### PR TITLE
[WIP] fix(settings): #4982 clear inputs on ESC

### PR DESCRIPTION
**Pull request by mutdmour**

### CODE REVIEW

**Comment by mutdmour** [comment]
@vladikoff  how do I run the mocha and chai unit tests?

**Comment by mutdmour** [comment]
@shane-tomlinson @vladikoff I am leaving for a two week vacation without a laptop. If anyone wants to finish this, feel free. Else, I will pick this back up when I am back. Feature works manually, but could not get the functional tests to work.

**Comment by TDA** [comment]
Does this need to be squashed into one commit? @mutdmour

**Comment by mutdmour** [comment]
@TDA ya it got pretty messy, because I wanted circleci to run the tests. I just got Open Source access to sauce labs, so I it should get better from here if that works. 
I will squash them once I am done.

File: tests/functional/settings_secondary_emails.js [hunk 1]
```diff
@@ -317,6 +320,28 @@ define([
         .then(switchToWindow(1))
         .then(testElementExists(selectors.CONNECT_ANOTHER_DEVICE.HEADER))
         .then(closeCurrentWindow());
-    }
+    },
+    'add secondary email, hit Esc, input should be cleared': function () {
+      const TEXT = 'TEST';
+      return this.remote
+            // create user, verify, sign in, open settings page
```
Comment (vladikoff) [inline tests/functional/settings_secondary_emails.js]: @ mutdmour could you fix indentation in this test, compare to what you see above

Comment (vladikoff) [reply tests/functional/settings_secondary_emails.js]: @mutdmour ^

**Comment by vladikoff** [comment]
@mutdmour Great work! Let's fix indent and squash it all into a single commit. After that I will do a bit more testing and we can merge this!

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 5289c7b64e6382da832c39500176a21e9ee07378
  * Your commit message body contains a line that is longer than 100 characters

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

**Comment by mutdmour** [comment]
Well, I missed up the rebase (first time using it) and pushed with --force. I am going to close this PRr and replicate my work in a different PR.

**Pull request closed without merging**